package.json

```diff
@@ -7,6 +7,9 @@
 		"type": "git",
 		"url": "https://github.com/DoctorMcKay/node-steam-tradeoffer-manager.git"
 	},
+	"scripts": {
+		"lint": "./node_modules/.bin/eslint ."
```

Comment:
You don't need to change your path, npm automatically tries to run `./node_modules/.bin/eslint` if possible.